geoips/schema/products/list.yaml

```diff
@@ -10,4 +10,4 @@ properties:
         type: array
         items:
           type: object
-          $ref: products.specs.product
+          $ref: products.bases.product
```

Comment:
Were these the fixes you meant to put in 190?

Reply:
Yes, some of them.